sdk/keyvault/Azure.AspNetCore.DataProtection.Keys/src/AzureKeyVaultXmlDecryptor.cs

```diff
@@ -9,7 +9,7 @@
 using Microsoft.AspNetCore.DataProtection.XmlEncryption;
 using Microsoft.Extensions.DependencyInjection;
 
-namespace Azure.Security.KeyVault.Secrets.AspNetCore.DataProtection
+namespace Azure.AspNetCore.DataProtection.Keys
```

Comment:
Do users ever see any types in this namespace?  Should we just leave it `Azure.AspNetCore.DataProtection`?  I worry that `Keys` is too vague a name given that the whole feature is for storing keys.  Maybe the same goes for the assembly name/nuget package discovery - should they use `KeyVault` instead of `Keys`?  I don't feel strongly if everyone else likes these names.

Reply:
There are no public types in this namespace. Regarding the package/namespace name I would like to land on a consistent naming pattern. Maybe we should've done `Azure.AspNetCore.DataProtection.Storage.Blobs` for the storage adapter so we can do `KeyVault.Keys` here.